Makefile

```diff
@@ -8,6 +8,7 @@ NUM_WORKERS := $(or $(NUM_WORKERS),0)
 STORAGE_POOL_PATH := $(or $(STORAGE_POOL_PATH), $(PWD)/storage_pool)
 SSH_PUB_KEY := $(or $(SSH_PUB_KEY),$(shell cat ssh_key/key.pub))
 PULL_SECRET :=  $(or $(PULL_SECRET), $(shell if ! [ -z "${PULL_SECRET_FILE}" ];then cat ${PULL_SECRET_FILE};fi))
+ROUTE53_SECRET := $(or $(ROUTE53_SECRET), "")
```

Comment:
what will happened with it? how it will be use?
just to understand, don't see any line that uses it,

Reply:
oh sorry, the context is in the inventory PR: https://github.com/filanov/bm-inventory/pull/317

Reply:
got it